xml/x509_jarm.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<fingerprints matches="tls.jarm" protocol="x509" database_type="service">
+  <!--
+    Fingerprint based on https://github.com/salesforce/jarm
+  -->
+
+  <fingerprint pattern="^2ad2ad16d2ad2ad00042d42d000000332dc9cd7d90589195193c8bb05d84fa$">
+    <description>Tor relay</description>
+    <example>2ad2ad16d2ad2ad00042d42d000000332dc9cd7d90589195193c8bb05d84fa</example>
+    <param pos="0" name="service.product" value="Tor"/>
+    <param pos="0" name="service.vendor" value="Tor Project"/>
+    <param pos="0" name="service.component.cpe23" value="cpe:/a:torproject:tor:-"/>
+  </fingerprint>
+
+  <fingerprint pattern="^2ad2ad0002ad2ad0002ad2ad2ad2ad5bf44aec534289dfa8e33148b66cd6c3$">
+    <description>Synology NAS</description>
+    <example>2ad2ad0002ad2ad0002ad2ad2ad2ad5bf44aec534289dfa8e33148b66cd6c3</example>
+    <param pos="0" name="os.device" value="NAS"/>
+    <param pos="0" name="os.family" value="Linux"/>
+    <param pos="0" name="os.product" value="DSM"/>
+    <param pos="0" name="os.vendor" value="Synology"/>
+    <param pos="0" name="hw.vendor" value="Synology"/>
+    <param pos="0" name="hw.device" value="NAS"/>
+  </fingerprint>
+
+  <fingerprint pattern="^2ad2ad0002ad2ad00042d42d000000ad9bf51cc3f5a1e29eecb81d0c7b06eb$">
+    <description>Proxmox open-source virtualization platform</description>
+    <example>2ad2ad0002ad2ad00042d42d000000ad9bf51cc3f5a1e29eecb81d0c7b06eb</example>
+    <param pos="0" name="service.vendor" value="proxmox"/>
+    <param pos="0" name="service.product" value="proxmox"/>
+    <param pos="0" name="os.vendor" value="Proxmox"/>
+    <param pos="0" name="os.family" value="Linux"/>
+    <param pos="0" name="os.product" value="Proxmox"/>
+  </fingerprint>
+
+  <fingerprint pattern="^2ad2ad16d2ad2ad22c2ad2ad2ad2ad7e5e7dc6f569c9c16238278a408347ef$">
+    <description>Ubiquiti EdgeRouter</description>
+    <example>2ad2ad16d2ad2ad22c2ad2ad2ad2ad7e5e7dc6f569c9c16238278a408347ef</example>
+    <param pos="0" name="hw.vendor" value="Ubiquiti"/>
+    <param pos="0" name="hw.device" value="Router"/>
+    <param pos="0" name="os.vendor" value="Ubiquiti"/>
+    <param pos="0" name="os.family" value="Linux"/>
+    <param pos="0" name="os.device" value="Router"/>
+    <param pos="0" name="os.component.cpe23" value="cpe:/a:ui:er-x-sfp:-"/>
+  </fingerprint>
+
+  <fingerprint pattern="^21d14d00021d21d21c21d14d21d21d">
+    <description>UniFi-Video Controller</description>
```

Comment:
FYI, the NIST string for this product is `UniFi Video Controller`. If you set that as the value for `os.product` (and `hw.product`) it will allow the CPE logic to autogenerate the correct value.

https://nvd.nist.gov/products/cpe/search/results?namingFormat=2.3&keyword=ui+video+controller